distancetransform.py

```diff
@@ -0,0 +1,124 @@
+__doc__ = """\
+DistanceTransform
+===========
+
+**DistanceTransform** 
+Computes the distance transform of a binary image. The distance of each
+foreground pixel is computed to the nearest background pixel. The
+resulting image is then scaled so that the largest distance is 1. 
+
+
+============ ============ ===============
+Supports 2D? Supports 3D? Respects masks?
+============ ============ ===============
+YES          YES          NO
```

Comment:
Are we sure respects masks is no here?

Reply:
It only works on a binary image, and at the moment you can't add another image as a mask

Reply:
Ah, let me clarify! "Respects masks" in this context means if you make an image, then mask it in MaskImage, and then run this on it, will the mask get carried forward to the output image, yes or no?

Reply:
ah, ok i get it now, then yes you are right it does ok I will change that should be the last thing to do